angular-with-directus/how to integrate directus with Angular.md

```diff
@@ -0,0 +1,449 @@
+
+```html
+<div *ngIf="post">
+  <div style="width: 500px">
+    <img style="width: 100%" [src]="baseUrl+'assets/'+post.image"
+         alt="{{post.title}}">
+  </div>
+  <h2>{{post.title}}</h2>
+  <p>{{post.content}}</p>
+</div>
+```
+### Add a Method to Handle a Click on the Blog Posts
+In `src/app/component/posts/posts.component.ts` file add the following code.
+
+```ts
+  constructor(private router: Router) {}
+
+  goToPost(id: number){
+    this.router.navigate(['/blog', id]);
+  }
+```
+This method will redirect you to `/blog/id` using `Route` when you click on an post on the blog post listing. The `id` path variable will be associated with the clicked item.
+
+As a result, a post will be loaded dynamically depending on which post you click.
+
+### Add a Click Listener for the Blog Posts
+In `src/app/component/posts/posts.component.html` add the method you have created in the previous section in the following line.
+
+```ts
+<a routerLink="#" (click)="goToPost(post.id)">
+      <h2>{{post.title}}</h2>
+</a>
+```
+Since the method expects the `id` parameter, pass `post.id` as the argument of the method. As a result, this will bind the method with the current id of a post at runtime.
+
+### Add Routing for the Blog Post Page
+In `src/app/app.routes.ts` file add the following route in the `Routes` array:
+
+```ts
+{path: 'blog/:id', component: PostComponent}
+```
+Once the application reloads, go to `http://localhost:4200/blog` and click on a post. As a result, the individual post will be displayed on the page via the path `http://localhost:4200/blog/id`.
+
+![blog post pages](blog-post-page.png)
+
+## Add Navigation
+To avoid navigating through the application manually, you should add navigation links that will dynamically route you to different components. To achieve this, go to `src/app/app.component.html` and add the following code before the `<router-outlet/>` tag.
+
+```html
+<nav>
+  <ul>
+    <li><a routerLink="/">Home</a> </li>
+    <li><a routerLink="/about">About</a> </li>
+    <li><a routerLink="/conduct">Code of Conduct</a> </li>
+    <li><a routerLink="/privacy">Privacy Policy</a> </li>
+    <li><a routerLink="/blog">Blog</a> </li>
+  </ul>
+</nav>
+```
+
+## Recap
+In this tutorial, you have learned how to integrate directus with Angular. You have covered how to use global metadata and settings, how to create pages, how to create a post listing, how to show blog post pages, and lastly how to add navigation in your application.
+
+The Directus CMS is quite diverse and it can accomodate any type of blog that you might want to create. For instance, one of the features of Directus you can leverage is authentication of requests. The Directus SDK makes authentication very easy for you. Go to the authentication [guide](https://docs.directus.io/guides/sdk/authentication.html) to learn how to authenticate your requests.
```

Comment:
Remove this - the previous paragraph is a perfect end.

Reply:
Okay